python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -83,6 +85,174 @@ def _apply_rotary_emb(
         return torch.stack((o1, o2), dim=-1).flatten(-2)
 
 
+# Adapted from https://github.com/linkedin/Liger-Kernel/blob/v0.6.2/src/liger_kernel/ops/qwen2vl_mrope.py
+
+_mrope_forward_autotune = triton.autotune(
```

Comment:
We normally don't use triton.autotune in sglang, fp8_autotune and fused_softcap_autotune are two exceptions.